React/Libraries.md

```diff
@@ -0,0 +1,53 @@
+#### React Libraries
+
+##### React Development Tools
+ - [create-react-app](https://github.com/facebook/create-react-app) - Project setup by running one command
+ - [storybook](https://github.com/storybookjs/storybook) - Development environment for UI components
+
+##### React Frameworks
+- [next.js](https://github.com/vercel/next.js) - The React Framework
+
+##### React Routing
+- [next-connect](https://github.com/hoangvvo/next-connect) - Minimal router and middleware layer for Next.js
+- [react-router](https://github.com/ReactTraining/react-router) - Declarative routing for React
+
+##### React Styling and Component Libraries
```

Comment:
this should really only include theme-ui or chakra,

Reply:
Out of 4 projects I looked into, 2 used `theme-ui`, 2 used `emotion` (for theming), and 1 used `ant-design`. One also used `styled-system` which I'm not familiar with.
I added `chakra-ui`, and also `styled-components` because we use it with React Native projects (guessing that will go into another list).

Reply:
ChakraUI and ThemeUI depend on emotion and styled-system.
I would mention only ChakraUI (ThemeUi could be migrated to Chakra fairly easily because they have the same API)
Maybe we should consider leaving `emotions` and `styled-system` just for reference and mention that Chakra is using those libs under the hood.

Reply:
We should also keep a reference to Ant Design because we will probably use it on some projects where the design is not that important. This should of course be noted that it should not be used unless we have an agreement beforehand.

Reply:
Added a note `(Used by ChakraUI)` for `emotion` and `styled-system`.
Should we have a category like `Used, but not recommended` for Ant Design? (maybe also put some other libs there)

Reply:
> Added a note (Used by ChakraUI) for emotion and styled-system.
Should we have a category like Used, but not recommended for Ant Design? (maybe also put some other libs there)

I'm ok with that.
@DarkoKukovec what do you think?

Reply:
Yep, sounds fine